modules/core/src/test/java/org/apache/ignite/internal/processors/cache/persistence/snapshot/IgniteSnapshotManagerSelfTest.java

```diff
@@ -581,6 +581,40 @@ public void testSnapshotThreadPoolSizeUsage() throws Exception {
         assertEquals(snapshotThreadPoolSize.longValue(), snpRunningThreads);
     }
 
+    /**
+     * Tests that full-copy snapshot logs correctly.
+     *
+     * @throws Exception If fails.
+     * */
+    @Test
+    public void testSnapshotCreationLog() throws Exception {
+        if (listenLog == null)
+            listenLog = new ListeningTestLogger(log);
+
+        final int ENTRIES_CNT = 4;
```

Comment:
ENTRIES_CNT -> entriesCnt. Use upper case for class-level constants only [1]

[1] https://cwiki.apache.org/confluence/display/IGNITE/Coding+Guidelines#CodingGuidelines-Naming